packages/components/src/mobile/inserter-button/index.native.js

```diff
@@ -16,6 +16,11 @@ import { __ } from '@wordpress/i18n';
  */
 import styles from './style.scss';
 
+export const inserterButtonStyles = {
```

Comment:
What do you think if instead of exporting the styles separately we do something like [we do for the BottomSheet ](https://github.com/WordPress/gutenberg/blob/master/packages/components/src/mobile/bottom-sheet/index.native.js#L466-L478)?

So we have something like:

```
const InserterButton = withPreferredColorScheme( MenuItem );

InserterButton.Styles = inserterButtonStyles;

export default InserterButton;
```

Reply:
Yeah I like that pattern. Lets go with that! Thanks for pointing it out!